pandas/tests/arrays/test_integer.py

```diff
@@ -710,6 +710,21 @@ def test_to_integer_array_float():
     assert result.dtype == Int64Dtype()
 
 
+@pytest.mark.parametrize(
+    "values, dtype",
+    [
+        ([9999999999999999, 123123123123123123, 10000000000000543, np.nan], "Int64"),
+        (np.array([9999999999999999, 123123123123123123, 10000000000000543],
+         dtype=np.int64), "Int64"),
+    ],
+)
+def test_to_integer_array_large_numbers(values, dtype):
+    result = integer_array(values, dtype=dtype)
+    expected = integer_array(values, dtype="Int64")
```

Comment:
does this test fail on master?

I _think_ you may need to construct the EA explicitly, i.e.

`pd.Series(IntegerArray(np.array([1, 2, 3, x, y, z, 0], dtype="int64"), np.array([False, False, False, False, False, False, True])))`

could use 
```
expected = pd.Series([1, 2, 3, x, y, z, 0], dtype="Int64")
expected[6] = np.nan
```

but i'd prefer the explicit approach.

Reply:
No it's not failing on master.